### PR TITLE
Blogs - Multi-Cluster monitoring with Prometheus and Submariner

### PR DESCRIPTION
**Pull request by danibachar**
Signed-off-by: I <danibachar89@gmail.com>

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

File: src/content/other-resources/_index.en.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ There are multiple presentations and demo recordings about Submariner available
 
 ## Blogs
 
+* [Multi-Cluster monitoring with Prometheus and Submariner (9/2021)](https://medium.com/@danielbachar/multi-cluster-monitoring-with-prometheus-and-submariner-f89ff733e7ec)
```
Comment (nyechiel) [inline src/content/other-resources/_index.en.md]:
```suggestion
* [Multi-Cluster monitoring with Prometheus and Submariner (09/2021)](https://medium.com/@danielbachar/multi-cluster-monitoring-with-prometheus-and-submariner-f89ff733e7ec)
```

**Review by nyechiel — COMMENTED** [review]
One nit to keep a consistent writing style

**Review by nyechiel — APPROVED** [review]

File: src/content/other-resources/_index.en.md (hunk 1 above)
Comment (danibachar) [reply src/content/other-resources/_index.en.md]: Fixed

**Comment by nyechiel** [comment]
@skitt @sridhargaddam can you please take a look at this? I want to push some changes to the same file and prefer this one to be merged first. Thanks!

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**